map/local_ads_manager.cpp

```diff
@@ -719,21 +710,22 @@ LocalAdsManager::FeaturesCache LocalAdsManager::ReadCampaignFeatures(CampaignDat
 
   auto const deviceLang = StringUtf8Multilang::GetLangIndex(languages::GetCurrentNorm());
   m_readFeaturesFn(
-    [&campaignData, &cache, deviceLang](FeatureType & ft) {
-      auto it = campaignData.find(ft.GetID());
-      CHECK(it != campaignData.end(), ());
-
-      LocalAdsManager::CacheEntry entry;
-      entry.m_position = feature::GetCenter(ft, scales::GetUpperScale());
-      entry.m_isVisibleOnMap = it->second != nullptr;
-      if (it->second != nullptr)
-      {
-        it->second->m_position = entry.m_position;
-        ft.GetPreferredNames(true /* allowTranslit */, deviceLang, it->second->m_mainText, it->second->m_auxText);
-      }
-      cache.insert(std::make_pair(it->first, entry));
-    },
-    features);
+      [&campaignData, &cache, deviceLang](FeatureType & ft) {
+        auto it = campaignData.find(ft.GetID());
+        CHECK(it != campaignData.end(), ());
+
+        LocalAdsManager::CacheEntry entry;
+        entry.m_position = feature::GetCenter(ft, scales::GetUpperScale());
+        entry.m_isVisibleOnMap = it->second != nullptr;
+        if (it->second != nullptr)
+        {
+          it->second->m_position = entry.m_position;
+          ft.GetPreferredNames(true /* allowTranslit */, deviceLang, it->second->m_mainText,
+                               it->second->m_auxText);
+        }
+        cache.insert(std::make_pair(it->first, entry));
+      },
+      features);
```

Comment:
Revert this part, please. It's big enough and does not concern the PR.
Or maybe revert the whole file.

Reply:
revert done